buildSrc/src/main/java/com/uber/okbuck/composer/android/AndroidBuckRuleComposer.java

```diff
@@ -39,15 +39,6 @@ public static String bin(AndroidAppTarget target) {
     return "bin_" + target.getName();
   }
 
-  /**
```

Comment:
Some change in recent versions of okbuck turned the name of the rule to `bin_debug_test`. Removing this fixes it of the `tests = []` field of its counterpart library (was `lib_instrumentation_debug_test_apk` before)